dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/QueueServiceImpl.java

```diff
@@ -64,6 +65,28 @@ public class QueueServiceImpl extends BaseServiceImpl implements QueueService {
     @Autowired
     private UserMapper userMapper;
 
+    /**
+     * Valid both queue and queueName when we want to create or update queue object
+     *
+     * @param queue queue value
+     * @param queueName queue name
+     * @return Optional of Status map
+     */
+    private Optional<Map<String, Object>> queueValid(String queue, String queueName) {
```

Comment:
For reviewers, I add `valid` for both `queue` and `tenant` create or update process, to combine the same logic from each function